service/controller/v15/resource/s3object/create.go

```diff
@@ -33,34 +33,41 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 				return microerror.Mask(err)
 			}
 
-			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating S3 object '%s': created", key))
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("created S3 object '%s'", key))
 		} else {
-			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating S3 object '%s': already created", key))
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not create S3 object '%s'", key))
```

Comment:
I think reason log would be useful here. Sth `S3 object '%s' already exists` would work for me.

Reply:
Done.